public/HTML/index.html

```diff
@@ -0,0 +1,251 @@
+        </div>
+      </div>
+    </div>
+<!--Side bar-->
+    <div class="side-bar">
+      <div class="sideBarPost"></div>
+      <div class="dates">
+
+      </div>
+      <div class="date-post">
+        <button class="greyButton"> March 21, 2020</button>
+        <br>
+        <h6><a href="blog-post.html"> JUST ANOTHER POST</a></h6>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi doloribus aspernatur culpa quo quod reiciendis
+          id magni? Quam, error ducimus!</p>
+          <hr>
+          <br>
+      </div>
+      <div class="sideBarPost"></div>
+      <div class="dates">
+
+      </div>
+      <div class="date-post">
+        <button class="greyButton"> April 7, 2020</button>
+        <br>
+        <h6><a href="blog-post.html"> JUST ANOTHER POST</a></h6>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit exercitationem placeat culpa obcaecati aut
+          laborum pariatur soluta fugit totam accusantium.</p>
+          <hr>
+          <br>
+        <div class="sideBarPost"></div>
+        <div class="dates">
+
+        </div>
+        <div class="date-post">
+          <button class="greyButton"> May 17,2020</button>
+          <h6><a href="blog-post.html"> JUST ANOTHER POST</a></h6>
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Architecto optio repellendus deleniti atque non
+            saepe provident quod tempora libero excepturi.</p>
+          <hr>
+          <hr>
+        </div>
+
+        <div class="highlight-post">
+          <div class="hightPost">
+            <div class="sidebarTitles">
+              <br>
+              <h5> SOMETHING OF NOTE</h5>
+
+              <div class="imgTextWrapper">
+                <div class="squareImageWrapper grey-border"></div>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iste deleniti nobis unde ut ad officiis
+                  quasi, libero delectus distinctio eaque?</p>
+                <button class="redButton"><a href="learn-more.html">LEARN MORE</button>
+                <br>
+                <hr>
+                <br>
+              </div>
+            </div>
+          </div>
+
+          <div class="highlight-post">
+            <div class="hightPost">
+              <div class="sidebarTitles">
+                <h5> HIGHLIGHT</h5>
+                <div class="imgTextWrapper">
+                  <div class="squareImageWrapper grey-border"></div>
+                  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Perspiciatis totam distinctio obcaecati
+                    incidunt fugit ipsam nam odit minima suscipit deserunt?</p>
+                    <br>
+                  <button class="redButton"><a href="learn-more.html">LEARN MORE</button>
+                  <br>
+                  <hr>
+                  <br>
+                </div>
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+
+    </div>
+

```

Comment:
Your sidebar looks great, in terms of overall layout.  Internally, however, it could stand some fine-tuning!  There is no buffering between individual headers, titles, image-placeholder `<div>`s, buttons, or the borders below those buttons.  Just some fine tuning, but internal formatting is important too!